frontend/packages/dev-console/integration-tests/support/commands/app.ts

```diff
@@ -0,0 +1,59 @@
+export {}; // needed in files which don't have an import to trigger ES6 module usage
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      titleShouldBe(title: string): Chainable<Element>;
+      alertTitleShouldBe(title: string): Chainable<Element>;
+      selectLinkInBreadCrumb(linkName: string): Chainable<Element>;
+      selectKebabMenuOption(kebabMenuOption: string): Chainable<Element>;
+      selectActionsMenuOption(actionsMenuOption: string): Chainable<Element>;
+    }
+  }
+}
+
+Cypress.Commands.add('titleShouldBe', (title: string) => {
```

Comment:
Agreed Dave, even I am planning to merge the selectors with console, so that every one can re-use it. so there won't be necessity of re-inventing the wheel. I will update it as **pageTitleShouldContain** will this work? 
The reason I used titleShouldBe is most of the dev-console ui pages are using same locator.

Reply:
Hi @makambalaji, so are you saying you will be refactoring and using [admin detail page helper](https://github.com/openshift/console/blob/5150fde959976b79608956031a555c0bbef01598/frontend/packages/integration-tests-cypress/views/details-page.ts#L2)?

Reply:
yeah, Refactoring will do, where ever applicable. after a couple of PR's merge